Tests/SwiftBasicFormatTest/BasicFormatTests.swift

```diff
@@ -341,4 +341,22 @@ final class BasicFormatTest: XCTestCase {
         """
     )
   }
+
+  func testNodeHavingParentNode() {
+    let decl: DeclSyntax = """
+      func test() {
+        print(1)
+      }
```

Comment:
I just realized that there is a simpler and more intuitive example for this behavior:

```suggestion
    func test() {
        print(1)
    }
```

Reply:
Fixed it 👍 [9d40557](https://github.com/apple/swift-syntax/pull/1791/commits/9d4055702bf696227c66c9bf64152a580cafce45)